packages/core/src/components/DateRangePicker/DateRangePicker.stories.mdx

```diff
@@ -64,9 +64,10 @@ return <DateRangePicker value={dates} onChange={setDates} />;
     </Story>
 </Preview>
 
-### onPopupClose
+## onPopupClose
+
+You can pass a function using the `onPopupClose` property to perform any action after the calendar popup closes. To make `dateRangePicker` a controlled component, you need the `onChange` prop, and the `onPopupClose` prop handles actions like API calls after the date selection has been made.
```

Comment:
`To make `dateRangePicker` a controlled component, you need the `onChange` prop` this sentence is completely different from its connecting one. So I guess we can separate it from the connected sentence.

Reply:
Keeping the earlier sentence then